fetch.bs

```diff
@@ -3061,6 +3251,31 @@ in addition to <a>HTTP fetch</a> above.
   nullity has already been checked. The <a lt=extract for=BodyInit>extracting</a> operation cannot
   throw as it was called for the same <a for=body>source</a> before.
 
+ <li><p>Let <var>useReferredTokenBinding</var> be the result of
+   <a for=header>parsing</a> `<code>Include-Referred-Token-Binding-ID</code>`
+   in <var>actualResponse</var>'s <a for=response>header list</a>.
+
+ <li><p>Unset the <var>request</var>'s <a for=request>use-referred-token-binding flag</a>.
+
+ <li><p>If <var>useReferredTokenBinding</var> is a <span>byte-case-insensitive</span>
+     match for `<code>true</code>`, and the <var>request</var>'s
+     <a for=request>use-token-binding flag</a> is set, then perform the following substeps:
+
+ <ol>
+   <li><p>Set the <var>request</var>'s <a for=request>use-referred-token-binding</a> flag.
+   <li><p>Set the <var>request</var>'s <a for=request>referred-token-binding-origin</span> to
+   <var>request</var>'s <a for=request>origin</a>.
```

Comment:
This is the origin of the original page that generated the request (e.g. A in the redirect chain `A -> B -> C -> D`). It's not clear to me that it's a good idea to allow a third-party to determine whether that ID should be exposed.